package.json

```diff
@@ -0,0 +1,24 @@
+{
+  "name": "epam-jsa-black-pearl",
+  "version": "1.0.0",
+  "description": "Black Pearl workspace -- Chris Alex Alfred",
+  "main": "index.js",
+  "scripts": {
+    "test": "echo \"Error: no test specified\" && exit 1",
+    "lint": "./node_modules/.bin/eslint index.js"
```

Comment:
You can just use `eslint` here, node will take care of the rest.
It should also check all the javascript files not only the `index.js`

Reply:
remove uncessary code